packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -175,4 +183,24 @@ export default abstract class EasCommand extends Command {
     await this.analytics.flushAsync();
     return super.finally(err);
   }
+
+  protected override catch(err: Error): Promise<any> {
+    let baseMessage: string = '';
+    if (err instanceof EasCommandError) {
+      Log.error(err.message);
+      baseMessage = this.baseErrorMessage || `${this.id} command failed.`;
+    } else {
+      const cleanMessage =
+        err instanceof CombinedError && err?.graphQLErrors
+          ? err.message.replace('[GraphQL] ', '')
+          : err.message;
+      Log.error(cleanMessage);
+      baseMessage =
+        err instanceof CombinedError && err?.graphQLErrors
+          ? this.baseGraphQLErrorMessage
+          : this.baseErrorMessage || `${this.id} command failed.`;
```

Comment:
```suggestion
          : this.baseErrorMessage ?? `${this.id} command failed.`;
```
but you have to accept my other suggestion first.